MCDataProducts/inc/GenId.hh

```diff
@@ -41,8 +41,9 @@ namespace mu2e {
       MARS, StoppedParticleReactionGun, bremElectronGun, muonicXRayGun, //30
       fromSimParticleStartPoint, fromSimParticleCompact, StoppedParticleG4Gun, //33
       CaloCalib, InFlightParticleSampler, muplusDecayGun, StoppedMuonXRayGammaRayGun, //37
-      cosmicCRY,  pbarFlat, fromAscii, ExternalRMC, InternalRMC, CeLeadingLog, cosmicCORSIKA, Mu2eX,//45
-      lastEnum //46
+      cosmicCRY,  pbarFlat, fromAscii, ExternalRMC, InternalRMC, CeLeadingLog, cosmicCORSIKA, //44
+      gammaPairProduction, Mu2eX,//46
```

Comment:
I agree, though I asked @sophiemiddleton if there were any existing datasets using `Mu2eX`, which there aren't, and so I thought this might be acceptable. There are existing SU2020 datasets already using `gammaPairProduction = 45`, so it seemed like the least conflict to insert this enum before `Mu2eX`. If this isn't allowed I'll move it, I was just hoping to avoid having to have flags or different branches in histogramming code to tell which enum version the sample was produced with.